test/auth_aws/test_auth_oidc.py

```diff
@@ -1,828 +0,0 @@
-# Copyright 2023-present MongoDB, Inc.
```

Comment:
Would you be able to rename this file in a separate commit to make this review easier?

Reply:
Done in #1378